eng/ci/templates/official/jobs/build-artifacts.yml

```
@@ -0,0 +1,193 @@
+            "ToolVersion": "1.0"
+          },
+          {
+            "KeyCode": "CP-230012",
+            "OperationCode": "SigntoolVerify",
+            "Parameters": {},
+            "ToolName": "sign",
+            "ToolVersion": "1.0"
+          }
+        ]
+
+  - task: DeleteFiles@1
+    displayName: Delete CodeSignSummary files
+    inputs:
+      contents: '**/CodeSignSummary-*.md'
+
+  - task: EsrpCodeSigning@2
+    displayName: Sign SDK Analyzers assemblies
+    inputs:
+      ConnectedServiceName: ESRP Service-internal
+      FolderPath: sdk\Sdk.Analyzers\bin\Release
+      Pattern: Microsoft.Azure.Functions.Worker.Sdk*.dll
+      signConfigType: inlineSignParams
+      inlineOperation: |
+        [
+          {
+            "KeyCode": "CP-230012",
+            "OperationCode": "SigntoolSign",
+            "Parameters": {
+              "OpusName": "Microsoft",
+              "OpusInfo": "http://www.microsoft.com",
+              "FileDigest": "/fd \"SHA256\"",
+              "PageHash": "/NPH",
+              "TimeStamp": "/tr \"http://rfc3161.gtm.corp.microsoft.com/TSS/HttpTspServer\" /td sha256"
+            },
+            "ToolName": "sign",
+            "ToolVersion": "1.0"
+          },
+          {
+            "KeyCode": "CP-230012",
+            "OperationCode": "SigntoolVerify",
+            "Parameters": {},
+            "ToolName": "sign",
+            "ToolVersion": "1.0"
+          }
+        ]
+
+  - task: DeleteFiles@1
+    displayName: Delete CodeSignSummary files
+    inputs:
+      contents: '**/CodeSignSummary-*.md'
+
+  - task: EsrpCodeSigning@2
+    displayName: Sign DotNetWorker assemblies
+    inputs:
+      ConnectedServiceName: ESRP Service-internal
+      FolderPath: src
+      Pattern: Microsoft.Azure.Functions.Worker*.dll
+      signConfigType: inlineSignParams
+      inlineOperation: |
+        [
+          {
+            "KeyCode": "CP-230012",
+            "OperationCode": "SigntoolSign",
+            "Parameters": {
+              "OpusName": "Microsoft",
+              "OpusInfo": "http://www.microsoft.com",
+              "FileDigest": "/fd \"SHA256\"",
+              "PageHash": "/NPH",
+              "TimeStamp": "/tr \"http://rfc3161.gtm.corp.microsoft.com/TSS/HttpTspServer\" /td sha256"
+            },
+            "ToolName": "sign",
+            "ToolVersion": "1.0"
+          },
+          {
+            "KeyCode": "CP-230012",
+            "OperationCode": "SigntoolVerify",
+            "Parameters": {},
+            "ToolName": "sign",
+            "ToolVersion": "1.0"
+          }
+        ]
+
+  - task: DeleteFiles@1
+    displayName: Delete CodeSignSummary files
+    inputs:
+      contents: '**/CodeSignSummary-*.md'
+
+  - task: DotNetCoreCLI@2
+    displayName: Pack SDK
+    inputs:
+      command: custom
+      custom: pack
+      arguments: '--no-build -c release -p:PackageOutputPath=$(System.DefaultWorkingDirectory)\out\pkg\release -p:BuildNumber=$(buildNumber)'
```

I think you can use:

```suggestion
      arguments: '--no-build -c release -o $(System.DefaultWorkingDirectory)\out\pkg\release -p:BuildNumber=$(buildNumber)'
```

Also, no need to copy `out\pkg` style here. You could pack directly to `$(Build.ArtifactStagingDirectory)/NugetPackages` and then sign inline there also.